README.md

```diff
@@ -12,42 +12,124 @@ You found an issue?<br>Tell us about it - <a href="https://github.com/storyblok/
 
 ## Installation
 
-Make sure you've node `>= 5.11.0` installed.
+Make sure you've node `>= 9.11.0` installed.
 
-```
+```sh
 $ npm i storyblok -g
 ```
 
 ## Commands
 
-### Usage to kickstart a boilerplate, fieldtype or theme
+### select
 
-```
+Usage to kickstart a boilerplate, fieldtype or theme
+
+```sh
 $ storyblok select
 ```
 
-### Download your space's components schema as json
+### pull-components
 
+Download your space's components schema as json
+
+```sh
+$ storyblok pull-components --space <SPACE_ID>
 ```
-$ storyblok pull-components --space={{your_space_id}}
+
+#### Options
+
+* `space`: your space id
+
+### push-components
+
+Push your components file to your/another space
+
+```sh
+$ storyblok push-components <SOURCE> --space <SPACE_ID>
 ```
 
-### Push your components file to your/another space
+#### Parameters
+
+* `source`: can be a URL or path to JSON file.
 
+Using an **URL**
+
+```sh
+$ storyblok push-components https://raw.githubusercontent.com/storyblok/nuxtdoc/master/seed.components.json --space 67819
 ```
-$ storyblok push-components --space={{your_dest_space_id}} {{path/url}}
+
+Using a **path** to file
+
+```sh
+$ storyblok push-components ./components.json --space 67819
 ```
 
-### Logout from the Storyblok cli
+#### Options
+
+* `space`: your space id
+
+### sync
+
+Sync components, folder, roles or stories between spaces
+
+$ storyblok sync --command <COMMAND> --source <SPACE_ID> --target <SPACE_ID>
+
+#### Options
+
+* `command`: a command to execute the syncronization. Available commands:
+  * `syncFolders`
```

Comment:
Can we simplify this to --type folders, components, ...? Shorter to write. `storyblok sync --type folders`?

Possibility to provide multiple types would also be nice `storyblok sync --type folders components`.

Reply:
Already implemented. Thanks for the suggestion